zebra-state/src/service/finalized_state.rs

```diff
@@ -373,12 +373,30 @@ impl FinalizedState {
             let block_time = block.header.time.timestamp();
             let local_time = chrono::Utc::now().timestamp();
 
-            const AWAY_FROM_TIP_BULK_SIZE: usize = 800;
+            // Mainnet bulk size is small enough to avoid the elasticsearch 100mb content
+            // length limitation. MAX_BLOCK_BYTES = 2M. Each block count as 2 as we send
+            // them with a operation/header line. A value of 100 is 50 blocks.
+            const MAINNET_AWAY_FROM_TIP_BULK_SIZE: usize = 100;
+
+            // Testnet bulk size is larger as blocks are generally smaller in the testnet.
+            // A value of 800 is 400 blocks as we are not counting the operation line.
+            const TESTNET_AWAY_FROM_TIP_BULK_SIZE: usize = 800;
```

Comment:
This could break if someone adds 50 large blocks to testnet, which could happen at any time.

```suggestion
            const TESTNET_AWAY_FROM_TIP_BULK_SIZE: usize = 45;
```

Reply:
I will prefer to don't make changers now in the testnet as it was working. This change affects performance.

Reply:
No worries, I opened #7270 so we remember to fix this security issue before we stabilise this feature in production.